benches/benches/block_target_gas_set/other.rs

```diff
@@ -37,5 +37,23 @@ pub fn run_other(group: &mut BenchmarkGroup<WallTime>) {
             .run(id, group, instructions, script_data.clone());
     }
 
-    // gtf: TODO: As part of parent issue (https://github.com/FuelLabs/fuel-core/issues/1386)
+    // gtf
+    {
+        let count = 254;
```

Comment:
It would be nice if we updated the corresponding benchmark to get the right price for GTF=)

Reply:
Do you mean using `GTFArgs::InputContractOutputIndex` instead of `GTFArgs::ScriptData` in the `vm_set` benchmark? So they are testing the same argument for `gtf`?

Reply:
Yes, because we need to use the price for the GTF worst case. The getter for script data is fast